deployments/data102/image/environment.yml

```diff
@@ -0,0 +1,84 @@
+name: data102-sp23
+
+dependencies:
+- nodejs=15.*
+- pip=22.2.*
+- python=3.9.*
+
+- syncthing==1.18.6
+
+# pymc3 needs this
+- mkl-service=2.4.*
+
+# Base scientific packages that other conda packages we install depend on
+# We don't want to have conda packages depend on pip packages if possible
+- numpy=1.21.*
+- matplotlib=3.5.*
+- scipy=1.7.*
+- ipympl=0.9.*
+- pandas=1.3.*
+- statsmodels=0.13.5
+- scikit-learn=1.1.1
+- seaborn=0.11.*
+- bokeh=2.3.*
+- decorator=5.0.*
+- networkx=2.6.*
+- spacy=3.4.1
+- nltk=3.6.*
+
+# data 88e, fall 2022, issue 3915
```

Comment:
I'd remove all dependencies that are associated with courses other than data102. If the course does need one of them later it can be re-added. This helps to keep the image smaller, and reduces the odds you'll have to chase down dependency conflicts should data102 need an update to something.